LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 The Fortran Programming Language
```

Comment:
Would the copyright be better to reference specific _people_ rather than the language itself?

e.g.: 
```suggestion
Copyright (c) 2022 Fortran-Lang Contributors
```

Reply:
I was thinking more along the lines of:
**if** a person is submitting their work in this repo **and** they don't want to license and copyright their work separately the copyright should default to the GitHub Organisation. Basically this should act as a fallback license.

Using the Organisation vs the people in it, shifts any liability to the org not the individuals (at least that was my thinking for this). If we join NumFOCUS and something were to go, for example copyright infringement, **I would guess** that our legal counsel would have an easier time dealing with the org registered in NumFOCUS as opposed to all/some members of the org.

Either way you got a point, writing "The Fortran Programming Language" could easily be misconstrued if we want to refer to the GitHub Organisation it should be "Fortran-Lang"

Reply:
Ah I see, makes sense 👍 . Yeah, in that case, I think "Fortran-Lang" makes more sense that "Fortran Language".